src/utils/parseJob.js

```diff
@@ -51,6 +51,14 @@ export const parseJob = (job, tab) => {
       }
     }
   } else {
+    const filteredContextParameter = job.spec?.parameters
+      ? Object.fromEntries(
+          Object.entries(job.spec.parameters).filter(([key, value]) =>
+            key !== 'context' ? key : value
+          )
+        )
+      : {}
+
```

Comment:
```suggestion
let jobParameters = job.spec?.parameters || {}

if ('context' in jobParameters && !jobParameters.context) delete jobParameters.context 

```

Reply:
No need to parse the object and filter out the array. You can just delete the `context` from the object